Gemfile.lock

```diff
@@ -254,6 +254,7 @@ GEM
 
 PLATFORMS
   x86_64-darwin-21
+  x86_64-linux
```

Comment:
This is the same fix for the `prepare_next_version` fastlane job as in: https://github.com/RevenueCat/cordova-plugin-purchases/pull/189